src/alert/demos/enUS/index.demo-entry.md

```diff
@@ -15,17 +15,17 @@ no-icon
 
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-| closable | `boolean` | `false` |  |
-| show-icon | `boolean` | `true` |  |
-| title | `string` | `undefined` |  |
-| type | `'default' \| 'info' \| 'success' \| 'warning' \| 'error'` | `'default'` |  |
-| on-after-leave | `Function` | `undefined` |  |
-| on-close | `() => boolean \| Promise<boolean> \| any` | `() => true` |  |
+| closable | `boolean` | `false` | Can the alert information be turned off. |
```

Comment:
_Alert information_ could be confused with the actual alert as a component, I would rephrase it like: _Determines whether the alert can be closed/dismissed_